build/goreleaser/linux/debian_systemd_amd64.yml

```diff
@@ -6,7 +6,7 @@
       - linux-ctl-amd64
       - linux-service-amd64
     package_name: newrelic-infra
-    file_name_template: "newrelic-infra_systemd_{{ .Env.TAG }}_{{ .Arch }}"
+    file_name_template: "newrelic-infra_systemd_{{ .Env.TAG }}_{{ .Arch }}{{ .Env.FIPS }}"
```

Comment:
Same, this should be :

`"newrelic-infra_systemd{{ .Env.FIPS }}_{{ .Env.TAG }}_{{ .Arch }}"`